perception/navigator_vision/object_classification/lidar_to_image.py

```diff
@@ -0,0 +1,214 @@
+import txros
+from twisted.internet import defer
+from txros import util, tf
+import navigator_tools as nt
+from navigator_tools import Debug
+import sys
+from collections import deque
+from cv_bridge import CvBridge
+from nav_msgs.msg import Odometry
+from sensor_msgs.msg import Image, CameraInfo
+import genpy
+import cv2
+import navigator_tools as nt
+from navigator_msgs.srv import ObjectDBQuery, ObjectDBQueryRequest
+from image_geometry import PinholeCameraModel
+import numpy as np
+
+
+class LidarToImage(object):
+
+    def __init__(self, nh, training=False, classes=None, dist=50):
+        self.MAX_SIZE = 74
+        self.IMAGE_SIZE = 100
+        self.max_dist = dist
+        self.bridge = CvBridge()
+        self.nh = nh
+        self.id_to_perist = {}
+        self.image_cache = deque()
+        self.pose = None
+        self.training = training
+        self.image = None
+        self.classes = classes
+        self.cam_info = None
+        self.busy = False
+        self.c = 0
+
+        self.debug = Debug(nh)
+
+    @util.cancellableInlineCallbacks
+    def init_(self, right=True):
+        image_sub = "/stereo/right/image_rect_color"
```

Comment:
I would rather use params passed into the function. 

I personally, only thing ros params should be used in between nodes..